tests/tests_twobody/test_propagation.py

```diff
@@ -448,3 +448,17 @@ def test_propagate_with_coe(propagator_coe):
     nu_final = propagator_coe(k, p, ecc, inc, raan, argp, nu, period)
 
     assert_quantity_allclose(nu_final, nu)
+
+
+def test_danby_with_zero_eccentricity():
```

Comment:
Why only `danby` here? Is it because we're testing an `if` branch? If so, I think we could test all propagators with zero eccentricity

Reply:
Yes, `danby`'s core method had an `if ecc == 1.0` condition which wasn't checked (line 840 [here](https://codecov.io/gh/poliastro/poliastro/src/main/src/poliastro/core/propagation/__init__.py)). But as you say, it would be better to check all propagators.